app-ios/Modules/Sources/FloorMap/SideEventRow.swift

```diff
@@ -0,0 +1,109 @@
+import Assets
+import Component
+import shared
+import SwiftUI
+import Theme
+
+struct SideEventRow: View {
+    var sideEvent: SideEvent
+
+    var body: some View {
+        HStack(alignment: .top, spacing: 12) {
+            VStack(alignment: .leading, spacing: 8) {
+                VStack(alignment: .leading, spacing: 4) {
+                    // Icon & Title
+                    HStack(alignment: .top, spacing: 8) {
+                        sideEvent.icon
+                            .renderingMode(.template)
+                            .frame(width: 16, height: 16)
+                            .foregroundStyle(sideEvent.iconColor)
+
+                        Text(sideEvent.title.currentLangTitle)
+                            .font(.system(size: 14, weight: .bold))
+                    }
+                    // Description
+                    Text(sideEvent.description_.currentLangTitle)
+                        .font(.system(size: 14, weight: .medium))
+                }
+                // Date
+                HStack(alignment: .center, spacing: 4) {
+                    Assets.Icons.accessTime.swiftUIImage
+                        .frame(width: 16, height: 16)
+
+                    Text(sideEvent.timeText.currentLangTitle)
+                        .font(.system(size: 12, weight: .medium))
+                }
+            }
+            .foregroundStyle(AssetColors.Surface.onSurface.swiftUIColor)
+
+            Spacer(minLength: 0)
+
+            // Image
+            if let imageLink = sideEvent.link {
+                CacheAsyncImage(url: URL(string: imageLink)!) { image in
+                    image.resizable()
+                } placeholder: {
+                    Color.gray
+                }
+                .frame(width: 88, height: 88)
+                .scaledToFill()
+                .clipShape(RoundedRectangle(cornerRadius: 12))
+                .overlay(
+                    RoundedRectangle(cornerRadius: 12)
+                        .stroke(AssetColors.Outline.outline.swiftUIColor, lineWidth: 1)
+                )
+            } else {
+                Color.gray
+                    .frame(width: 88, height: 88)
+                    .clipShape(RoundedRectangle(cornerRadius: 12))
+                    .overlay(
+                        RoundedRectangle(cornerRadius: 12)
+                            .stroke(AssetColors.Outline.outline.swiftUIColor, lineWidth: 1)
+                    )
```

Comment:
I think common modifiers can be replaced with single declaration with `Group` 👀 

```swift
Group {
  if {
    AnyView()
  } else {
    AnyView()
  }
}
.frame(width: 88, height: 88)
.clipShape(RoundedRectangle(cornerRadius: 12))
.overlay(
RoundedRectangle(cornerRadius: 12)
  .stroke(AssetColors.Outline.outline.swiftUIColor, lineWidth: 1)
)
```

Reply:
Thanks! I implemented here! ( https://github.com/DroidKaigi/conference-app-2023/pull/931/commits/ed88010ff15b434947d22439ac0f00f61486658f )